pydoctor/model.py

```diff
@@ -808,6 +808,7 @@ def introspectModule(self,
             module_full_name = f'{package.fullName()}.{module_name}'
 
         spec = importlib.util.spec_from_file_location(module_full_name, path)
+        assert spec is not None, f"Cannot find spec for module {module_full_name} at {path}"
```

Comment:
We should only use `assert` if we can we guarantee that the condition is always true. If we're not that sure, it would be better to raise an exception instead.

I think there is at least one unlikely but possible scenario in which this condition is not true: if a file is deleted or moved between the moment its name is discovered and the moment it is introspected.

Reply:
A RuntimeError is OK ?